scripts/setup.sh

```diff
@@ -185,6 +185,41 @@ if [ ! -f /var/goddard/setup.lock ]; then
 
 		fi
 
+		# cool so now we have the keys
+		while read tkey tdomain tport
+		do
+
+			# debug
+			echo "Sanity Check if $tkey is actually running"
+
+			# done
+			echo "{\"build\":\"busy\",\"process\":\"Making sure $tkey is actually running\",\"timestamp\":\"$( date +%s )\"}"  > /var/goddard/build.json
+
+			# post to server
+			curl -X POST -d @/var/goddard/build.json http://hub.goddard.unicore.io/report.json?uid=$(cat /var/goddard/node.json | jq -r '.uid') --header "Content-Type:application/json"
+
+			# get the running apps
+			running_app_container=$(docker ps | grep $tkey) || true
```

Comment:
@Johanndutoit  post this back to the hub server so we have a record of the "before" state.